draft-ietf-rats-corim.md

```diff
@@ -1121,18 +1122,22 @@ The Reference Values Triple has the following structure:
 
 The `reference-triple-record` has the following parameters:
 
-* `ref-env`: Search criterion that locates an Evidence environment that matches the reference environment.
-* `ref-claims`: Search criteria that locates the Evidence measurements that match the reference Claims.
+* `ref-env`: Reference Environment Identity of the Target Environment
+
+* `ref-claims`: One or more measurement claims for the Target Environment
 
 To process `reference-triple-record` both the `ref-env` and `ref-claims` criteria are compared with Evidence entries.
+First `ref-env` is used as a Search criterion to locate the Evidence environment that matches the reference environment.
+Subsequently, the `ref-claims` from this triple are used to match against the Evidence measurements for the matched environment.
 If the search criteria are satisfied, the matching entry is re-asserted, except with the Reference Value Provider's authority.
 By re-asserting Evidence using the RVP's authority, the Verifier can avoid mixing Reference Values (reference state) with Evidence (actual state).
 See {{-rats-endorsements}}.
 Re-asserted Evidence using RVP authority is said to be "corroborated".
 
 #### Endorsed Values Triple {#sec-comid-triple-endval}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/310
+An Endorsed Values triple provides additional Endorsements that are valid when a Target Environment has been verified against Reference Values or Evidence.
```

Comment:
I think we got rid of this wording before because there is no requirement of verifying anything.